### PR TITLE
data path fixed; requirement added

### PR DESCRIPTION
**Pull request by desh2608**
Fixed issues pointed out by @jtrmal

### CODE REVIEW

**Comment by jtrmal** [comment]
I think also this line:
https://github.com/waldo-seg/waldo/blob/ec46860e4b5b41af001c35de2728d74fc79ebf8c/egs/dsb2018/v1/local/scoring.py#L37

should be removed

File: egs/icdar2015/v1/run.sh [hunk 1]
```diff
@@ -13,14 +13,14 @@ stage=0
...
-# local/check_dependencies.sh
+local/check_dependencies.sh
 
 # train/validate split
 train_prop=0.9
 seed=0
 if [ $stage -le 0 ]; then
   # data preparation
-  local/prepare_data.sh $train_prop $seed /home/desh/Research/icdar/icdar_2015/sample/
+  local/prepare_data.sh $train_prop $seed
```
Comment (jtrmal) [inline egs/icdar2015/v1/run.sh]: IMO, the path should be passed as a parameter and hard-coded in egs/icdar2015/v1/local/image_utils_test.py

Comment (desh2608) [reply egs/icdar2015/v1/run.sh]: The path, if not default, can be passed as a parameter here. If no path is passed, it uses the default value specified in prepare_data.sh

**Comment by desh2608** [comment]
@jtrmal The dsb2018 is maintained by @YiwenShaoStephen, so maybe he could look into your suggestion in the scoring.py file.

**Comment by YiwenShaoStephen** [comment]
@jtrmal  Yes, it can be removed. It just gives me a sense of progress.

**Comment by jtrmal** [comment]
All right, but those are only printed during scoring (that doesn't take
that long) and without any additional information, so you will get only N
strangely looking lines without any additional info, i.e. you end up with
handful of screens of an incomprehensible code, so I think the utility is
close to zero (or maybe non-zero to you and zero to others).
y.

**Comment by jtrmal** [comment]
You understand that you don't write the code for yourself only, right?
y.

**Comment by YiwenShaoStephen** [comment]
Yeah... I mean Desh can delete it in next commit...

**Comment by jtrmal** [comment]
OK, great, thanks!
y.

**Comment by desh2608** [comment]
Done.

**Comment by danpovey** [comment]
assuming OK; will merge.

**Pull request merged**